ref/config/serverConfiguration.adoc

```diff
@@ -24,171 +18,134 @@ following example:
 </server>
 ----
 
-This example `server.xml` configures the server to support Java Server Pages 2.3,
-to listen to incoming traffic to localhost on port 9080, and to automatically
-expand WAR files when they are deployed.
+This `server.xml` file configures the server to do the following things:
 
-The `server.xml` is described in more detail <<server-xml,below>>. When
-discussing config for {projectName}, the term "server config" can be used to refer
-to all of the files that make up the server config, or specifically to the
-config held in XML files; it is usually clear in context and, if not, the
-phrase "server XML config" might be used to clarify. The next few sections
-describe each of the config files in the order they are processed.
+* Support the Java Server Pages 2.3 feature
```

Comment:
I think switching to bullets should only happen if we reference back to the server.xml snippet which asciidoc can do. It can add markers to the xml and cross reference making it clear what is referring to. Otherwise I think sticking with the more compact form is better because I don't think what the xml does is as important as what it looks like and I think using bullets focuses too much on what it does vs what it looks like.

Reply:
Updated